packages/gatsby-plugin-postcss/src/gatsby-node.js

```diff
@@ -15,22 +15,25 @@ const findCssRules = config =>
 
 exports.onCreateWebpackConfig = (
   { actions, stage, loaders, getConfig },
-  { cssLoaderOptions = {}, postCssPlugins, ...postcssOptions }
+  { cssLoaderOptions = {}, postCssPlugins, ...postcssLoaderOptions }
```

Comment:
I renamed this argument in order to not confuse it with the new `postcssOptions` that postcss-loader actually has now. Because the `sourceMap` option is still a sibling to `postcssOptions` in postcss-loader, we actually have to put the `plugins` property in another level of nesting within the actual object named `postcssOptions` and not what we had here previously.

As you can see below, the full hierarchy is now:

```javascript:
postcssLoaderOptions === {
  postcssOptions: {
    plugins: [],
  },
  sourceMap: true,
}
```